development/kops/run_sonobuoy.sh

```diff
@@ -38,10 +46,12 @@ rm -f LICENSE
 chmod 755 sonobuoy
 
 echo "Testing cluster $KOPS_STATE_STORE"
-export KOPS_FEATURE_FLAGS=SpecOverrideFlag
-kops set cluster "${KOPS_CLUSTER_NAME}" cluster.spec.nodePortAccess=0.0.0.0/0
 ./sonobuoy run --mode=certified-conformance --wait --kube-conformance-image k8s.gcr.io/conformance:v1.18.9
```

Comment:
instead of hardcoding v1.18.9 here, could we get that from an environment var? maybe [GIT_TAG](https://github.com/aws/eks-distro/blob/main/projects/kubernetes/kubernetes/Makefile#L5)

Reply:
We have another ticket for pulling these artifacts from s3 or ECR